app/businessrates/authorisation/auth/DefaultAuthConnector.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package businessrates.authorisation.auth
+
+import java.net.URL
+
+import javax.inject.{Inject, Named}
+import uk.gov.hmrc.auth.core.PlayAuthConnector
+import uk.gov.hmrc.http.HttpPost
+import uk.gov.hmrc.play.config.ServicesConfig
+
+class DefaultAuthConnector @Inject()(val http: HttpPost, val config: ServicesConfig)
```

Comment:
you dont need this you can get one from a module.

Reply:
Do you mean uk.gov.hmrc.play.bootstrap.auth.DefaultAuthConnector? This doesn't seem to be available. I'm guessing it's because the bootstrap library is old?? "uk.gov.hmrc" %% "microservice-bootstrap" % 10.4.0
Should we be upgrading this as well?